packages/expo-modules-autolinking/tsconfig.json

```diff
@@ -1,11 +1,10 @@
 {
   "include": ["./src"],
+  "extends": "@tsconfig/node12/tsconfig.json",
   "compilerOptions": {
     "outDir": "./build",
-    "target": "esnext",
     "module": "commonjs",
     "moduleResolution": "node",
-    "strict": true,
```

Comment:
the base config adds the strict and esModuleInterop config options so i just removed them here. https://www.npmjs.com/package/@tsconfig/node12